src/modules/poweraccent/PowerAccentKeyboardService/KeyboardListener.cpp

```diff
@@ -133,6 +133,14 @@ namespace winrt::PowerToys::PowerAccentKeyboardService::implementation
             m_prevForegroundAppExcl = { foregroundApp,
                                       find_app_name_in_path(processPath, m_settings.excludedApps) };
 
+            if (!m_prevForegroundAppExcl.second) // Check applications running under some other applications with window name
+            {
+                m_prevForegroundAppExcl = { foregroundApp,
+                                            check_excluded_app_with_title(foregroundApp, processPath, m_settings.excludedApps) };
+
+                Logger::debug(L"Application window name: {}", processPath);
```

Comment:
I didn't delete it specifically so that we can look at the logs if there is an error. But I can remove it if it is not appropriate.

Reply:
I feel like we'd better remove this log. It's not very informative, because when compatibility issue arises, people specify the app, so we'd know the necessary info anyway.